packages/cubejs-server-core/src/core/server.ts

```diff
@@ -712,6 +712,7 @@ export class CubejsServerCore {
         sqlCache: this.options.sqlCache,
         standalone: this.standalone,
         allowNodeRequire: options.allowNodeRequire,
+        fastReloadEnabled: getEnv('fastReloadEnabled'),
```

Comment:
Don't you want to do it like:
```suggestion
        fastReloadEnabled: options.fastReloadEnabled || getEnv('fastReloadEnabled'),
```
In case option from outside is set?